crates/mirror-mirror/Cargo.toml

```diff
@@ -27,7 +27,7 @@ ordered-float = { version = "3.4.0", default-features = false }
 serde = { version = "1.0.158", default-features = false, features = ["derive", "alloc"], optional = true }
 speedy = { version = "0.8.5", optional = true }
 syn = { version = "2.0", features = ["full", "parsing"], optional = true }
-glam = { version = "0.22", optional = true }
+glam = { version = ">= 0.22, <= 0.24", optional = true }
```

Comment:
`version = "0.22"` still allows version `0.24` so if someone depends on `mirror-mirror` they can still do `glam = "0.24"`.

So is this change about making sure we don't use a version higher than 0.24? 0.24 is also the latest version so is it also about future proofing?

Reply:
Docs says:
```
1.2.3  :=  >=1.2.3, <2.0.0
1.2    :=  >=1.2.0, <2.0.0
1      :=  >=1.0.0, <2.0.0
0.2.3  :=  >=0.2.3, <0.3.0
0.2    :=  >=0.2.0, <0.3.0
0.0.3  :=  >=0.0.3, <0.0.4
0.0    :=  >=0.0.0, <0.1.0
0      :=  >=0.0.0, <1.0.0
```

Reply:
lol yes sorry you're absolutely right 😜 Monday morning brain

Reply:
Next question then: Why is this a draft? I'm looking to do a release of mirror-mirror today so you want this included in that release?

Reply:
Anyway. The range thing.
I've done that for some other crates now.
Makes sense to support a range of glam versions like this for mirror-mirror as well?

Reply:
Yes.

Reply:
About the draft.
mirror-mirror also depends on macaw.
and macaw depends on glam. But I can't publish a new version of macaw until a new speedy has been published.
I guess I could just make mirror-mirror depend on a range of macaw releases? Or would that cause trouble?

Reply:
I don't know.

Reply:
For reference, speedy has been updated to allow glam 0.25, so closing this.

(macaw hasn't, but we don't need to update everything at the same time, as we now allow a version range).